### PR TITLE
Rework default Server.fs - move to a module

### PR DESCRIPTION
**Pull request by isaacabraham**
Small reformatting but also move from a class to a module and let-bound functions.

### CODE REVIEW

File: Content/default/src/Server/Server.fs [hunk 1]
```diff
@@ -6,38 +6,32 @@ open Saturn
 
 open Shared
 
-type Storage() =
-    let todos = ResizeArray<_>()
-
-    member __.GetTodos() = List.ofSeq todos
-
-    member __.AddTodo(todo: Todo) =
+module Storage =
```
Comment (theimowski) [inline Content/default/src/Server/Server.fs:9]: it looks more succinct and maybe even more idiomatic. I'm not convinced though - prefer to use explicit class/object notation when dealing with internal state. Probably a matter of preference though - curious what other think

Comment (isaacabraham) [reply Content/default/src/Server/Server.fs:9]: @theimowski in absence of any feedback (happy for you to loop in some people explicitly) - I would suggest we go with this one and add a recipe for the class-based (with DI) approach.

File: Content/default/src/Server/Server.fs [hunk 2]
```diff
@@ -6,38 +6,32 @@ open Saturn
...
 let todosApi =
-    { getTodos = fun () -> async { return storage.GetTodos() }
-      addTodo =
-          fun todo ->
-              async {
-                  match storage.AddTodo todo with
-                  | Ok () -> return todo
-                  | Error e -> return failwith e
-              } }
+    {
```
[the remark this answers is not in the document]
Comment (isaacabraham) [reply Content/default/src/Server/Server.fs:24]: Probably not.

**Comment by isaacabraham** [comment]
@theimowski I'm going to merge this in on Monday unless there's any real objections.

**Comment by theimowski** [comment]
Ok - feel free to do so 👍

sob., 13 lis 2021, 18:07 użytkownik Isaac Abraham ***@***.***>
napisał:

**Pull request merged**